test/apps/mapbox-tile/utils/decode.js

```diff
@@ -0,0 +1,41 @@
+import Protobuf from 'pbf';
+import {VectorTile} from '@mapbox/vector-tile';
+import {worldToLngLat} from 'viewport-mercator-project';
+import {vectorTileFeatureToGeoJSON} from './feature';
+
+const TILE_SIZE = 512;
+
+export function decodeTiles(x, y, z, arrayBuff) {
```

Comment:
Use full name for arguments

Reply:
What do you mean? Use `arrayBuffer` instead of `arrayBuff`?

Reply:
Yes